### DISCUSSION TITLE
The SED is  no longer in the child table of a Parent, the Parent still respond Data Request(802.15.4) with ACK(802.15.4). Is this behavior as expected?

### DISCUSSION THREAD

**Discussion by jinpeng1989** [openthread/openthread#8407] [Q&A] [answered]

**Comment by jwhui** [comment, marked as answer]
Yes, this is expected. IEEE 802.15.4 ACKs are generated at the MAC layer without any input from higher layers.

If a child wants to validate that it is still in its parent's child table, it needs to have a protocol exchange at a higher layer - for example, something that involves exchanging IPv6 datagrams.

**Comment by abtink** [comment]
As Jonathan said, yes, expected  behavior. 

You can check  "[Child Supervision](https://openthread.io/guides/build/features/child-supervision)" feature in OT as a mechansim related to this.

**Comment by jinpeng1989** [comment]
Thank you for your prompt response. I get it. Rx-off-when-idle Children SHOULD also periodically resynchronize by sending a Child Update Request to their Parent, then it can know if the Parent had timed out and attempt to reattach.
![image](https://user-images.githubusercontent.com/97089879/202370977-1aecfc9a-200e-4709-bd41-ce52210adb40.png)



By the way, ask a question about the Wireshark tool. Whether the Wireshark can parse the TLVs of Thread Management Framework(61631 port)? How to configure it?

**Reply by jwhui** [reply]
> By the way, ask a question about the Wireshark tool. Whether the Wireshark can parse the TLVs of Thread Management Framework(61631 port)? How to configure it?

Right-click on a packet, select "Decode As..." and select "CoAP-TMF".
